code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -17,7 +17,7 @@
 	var/list/activate_reagents = list() ///Reagents required for activation
 	var/recurring = FALSE
 	var/research ///Research point value for slime cores. These are defines stored in [/__DEFINES/research] - the actual values are updated there.
-
+	var/color_slime ///the color of the extract and the slime it came from
```

Comment:
How about a type path instead of a string?

Reply:
What do you mean by a type path? I used a string because the slime subtype use strings in their list to chose their color so I thought it was more appropriate.

Reply:
Yea, nevermind, this makes sense for slimes.